pkg/machine/keys.go

```diff
@@ -33,18 +33,16 @@ func CreateSSHKeys(writeLocation string) (string, error) {
 	return strings.TrimSuffix(string(b), "\n"), nil
 }
 
-func CreateSSHKeysPrefix(dir string, file string, passThru bool, skipExisting bool, prefix ...string) (string, error) {
-	location := filepath.Join(dir, file)
-
-	_, e := os.Stat(location)
+func CreateSSHKeysPrefix(identityPath string, passThru bool, skipExisting bool, prefix ...string) (string, error) {
```

Comment:
Does this need to be public?

Reply:
Yes. The reference can be found at: https://github.com/containers/podman/blob/249f0463ebf8fa1cc82fd4d345aca2f244cf5a5d/pkg/machine/wsl/machine.go#L975-L977